src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -163,5 +165,28 @@ public int MaxModelValidationErrors
         /// </ul>
         /// </remarks>
         public IList<IMetadataDetailsProvider> ModelMetadataDetailsProviders { get; }
+
+        /// <summary>
+        /// Provides programmatic configuration for the html helpers and view context.
+        /// </summary>
+        public HtmlHelperOptions HtmlHelperOptions
```

Comment:
cc @rynowak Would having this be a separate top level option be easier to deal with when it comes to layering?